TransferMethod/Sources/ListTransferMethodController.swift

```diff
@@ -91,20 +91,18 @@ final class ListTransferMethodController: UITableViewController {
     }
 
     override public func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        if let transferMethod = presenter.sectionData[indexPath.row].title {
-            showConfirmationAlert(title: "mobileAreYouSure".localized(),
-                                  message: String(format: "mobileRemoveEAconfirm".localized(), transferMethod),
-                                  transferMethodIndex: indexPath.row)
-            tableView.deselectRow(at: indexPath, animated: true)
-        }
+        showConfirmationAlert(title: "mobileRemoveEAconfirm".localized(),
+                              message: "mobileAreYouSure".localized(),
+                              transferMethodIndex: indexPath.row)
+        tableView.deselectRow(at: indexPath, animated: true)
     }
 
     override public func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-        return nil
+        nil
```

Comment:
Oh I think Vikas was also facing this issue wherein swiftlint was autocorrecting and removing return

Reply:
of that's probably from when xcode when I run build, revert?

Reply:
As long as it does not complain, then it's fine. But then it will be inconsistent with rest of the files.